modules/policy-criteria.adoc

```diff
@@ -330,25 +339,25 @@ Examples: +
 
 
 | Privileged container
-| Privileged running deployments.
+| Check if a deployment is configured in privileged mode. This criterion only checks against the value of the `privileged` field in the respective link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#securitycontext-v1-core[Pod Security Context].
```

Comment:
```suggestion
| Check if a deployment is configured in `privileged` mode. This criterion only checks the value of the `privileged` field in the respective link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#securitycontext-v1-core[Pod Security Context].
```
Can we link to the upstream Kubernetes doc??

Reply:
is `privileged` a mode or just a field?? need to maintain consistency for clarity.

Reply:
Privileged is a mode, which is configured by having a value of "true" in the "privileged" field, so I think this is OK?